tests/index.spec.ts

```diff
@@ -35,6 +35,11 @@ describe("addFormats options", () => {
     expect(validateTime("17:27:38Z")).toEqual(true)
     expect(validateTime("25:27:38Z")).toEqual(true)
     expect(validateTime("17:27")).toEqual(false)
+
+    const validateDatetime = ajv.compile({format: "date-time"})
+    expect(validateDatetime("2016-12-31T23:59:60Z")).toEqual(true)
+    expect(validateDatetime("2016-15-31T23:59:60Z")).toEqual(true)
+    expect(validateDatetime("2016-12-31 23:59:60Z")).toEqual(false)
```

Comment:
We should add the examples from #55...
```suggestion
    expect(validateDatetime("2016-12-31 23:59:60Z")).toEqual(false)
    expect(validateDatetime("2015-12-31t23:59:60Z")).toEqual(false)
    expect(validateDatetime("2015-02-11t22:59:22Z")).toEqual(true)
    expect(validateDatetime("2015-02-11t24:59:22Z")).toEqual(false)
    expect(validateDatetime("2015-02-11t22:59:22+24:30")).toEqual(false)
    expect(validateDatetime("2015-00-11t22:59:22+00:00")).toEqual(false)
    expect(validateDatetime("2015-01-00T22:59:22+00:00")).toEqual(false)